app/components/find/courses/qualifications_summary_component/view.html.erb

```diff
@@ -1,24 +1,24 @@
-<% case find_outcome %>
-<% when "QTS" %>
-  <%= govuk_details(summary_text: "QTS") do %>
+<% case course.qualification %>
+<% when 'qts' %>
```

Comment:
I changed the way it was done slightly. 

Before: it was using the translation to do a case for another translation.
After: Passing the course qualification and then translate.

I thought to split into partials but then I rollback and the code will be like 

View -> Component -> Another component -> Multiple partials